Makefile

```diff
@@ -0,0 +1,26 @@
+SHELL := /bin/bash
+
+.PHONY: tools lint build dependencies test
+
+tools:
+	# install gometalinter
+	go get -u github.com/alecthomas/gometalinter
+
+	# install gox
+	go get -v github.com/mitchellh/gox
```

Comment:
Just a note that you have this in but we don't actually use it. But you can leave it in since you'll probably integrate release on tag (which isn't in this PR yet, see https://github.com/Intellection/vault-init/pull/1#pullrequestreview-299177970).